backend/src/binaryInstaller.ts

```diff
@@ -40,76 +41,56 @@ class BinaryInstaller {
 
   async installBinary(installer: Installer) {
     return new Promise(async (resolve, reject) => {
-      var tmpDir = tmp.dirSync({ unsafeCleanup: true, keep: true })
-
-      // Stop and remove old binaries
-      await this.migrateBinaries(installer.binaryPath())
```

Comment:
Please add some cleanup of the old locations:
- Uninstall the remoteit agent
- Delete the old binaries